website/static/js/search.js

```diff
@@ -76,6 +76,7 @@ var ViewModel = function(params) {
     self.resultsPerPage = ko.observable(10);
     self.categories = ko.observableArray([]);
     self.searchStarted = ko.observable(false);
+    self.jsonData = ko.observable('');
```

Comment:
Why does this need to be an observable? Could we check to see if the "SHARE" category is present, instead?